src/containers/SearchForHelpPage/index.jsx

```diff
@@ -1,24 +1,80 @@
-import React from "react";
+import React, { useState } from "react";
```

Comment:
Out master always suggests to use `React.useState` rather than import it

Reply:
@louisrli what would you say? should i change it?

Reply:
i prefer React.useState, writing imports is annoying

Reply:
but its more important to be consistent in a codebase so whatever all the other components are doing, do thesame thing

Reply:
I checked Jundi's and Hiba's components, they imported useState too. so I think I should it leave it like this